calcom/cal.com
# feat: enable 'Overlay Calendar' enabled by default for signed in users

## What does this PR do?



- Fixes #18146(GitHub issue number)



Loom Video: https://www.loom.com/share/b67e6f4aa2ef4dcb9605cca4c28d74b9

## Mandatory Tasks (DO NOT REMOVE)


## How should this be tested?



- For signed in user, simply see if "Overlay Calendar" is enabled by default
- For un-signed user, simply see if "Overlay Calendar" is disabled by default
